active/003-seq-feature.md

```diff
@@ -126,12 +134,19 @@ Another alternative is to make `BoundFeature` objects that are tightly coupled w
 This will ease the process of updating and deleting interval/features from the `IntervalMetadata` object.
 
 ## `BoundFeature` object structure
-This object is a mutable object, that contains attributes (i.e. `name`, `function`, `strand`, ...) as well as a list of intervals.  This object would have a [weakref](https://docs.python.org/3/library/weakref.html) to the corresponding interval object in the `IntervalTree`.  So if the intervals within `BoundFeature` are updated, the intervals within the `IntervalTree` are updated.
+This object is a *mutable* object, that contains attributes (i.e. `name`, `function`, `strand`, ...) as well as a list of intervals.  This object would have a [weakref](https://docs.python.org/3/library/weakref.html) to the corresponding interval object in the `IntervalTree`.  if the intervals within `BoundFeature` are updated, the intervals within the `IntervalTree` are updated. We will probably want to have a reserved keyword for `intervals` in the `BoundFeature` object.  For instance, if we wanted to grab all of the intervals associated with a features, we should be able to run something as follows
+```python
+f = BoundFeature([1, (4, 7)], gene='sagA', function='toxin')
```

Comment:
I like this, but would it be possible to do something like:

```python
seq.interval_metadata[1, 4:7] = Feature(gene='sagA', function='toxin') 
```
which strikes me as a little more "pythonic" or at least pandas-y.

In this case `Feature` would act kind of like a stamp, resulting in `BoundFeatures`. Or perhaps `Feature` just has two states where the transition results in a new `Feature`? I'm not really sold on either idea, but I do prefer the above syntax.